docker-compose-generator/docker-fragments/bitcoin-eclair.yml

```diff
@@ -0,0 +1,53 @@
+version: "3"
+
+services:
+  bitcoind:
+    environment:
+      BITCOIN_WALLETDIR: "/data/wallet"
+      BITCOIN_EXTRA_ARGS: |
+        zmqpubrawblock=tcp://0.0.0.0:28332
+        zmqpubrawtx=tcp://0.0.0.0:28333
+        rpcauth=eclair:d3a244200d5c946f6bef48e6583ec979$215049513aef33245dfec8ff3ab5e749458fde51ede931dec04c767c0732d468
```

Comment:
Very subtle: there is a `$` sign causing docker-compose to fail parsing that, see [fix](https://github.com/araspitzu/btcpayserver-docker/commit/fb976a53dd4620d3593d99dce3a89cf280d9771d)

Reply:
Indeed, needed that too for lnd.

Reply:
fixed